test/integration/functional_test.go

```diff
@@ -1663,6 +1664,29 @@ func validateCertSync(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// validateNotActiveRuntimeDisabled asserts that for a given runtime, the other runtimes disabled, for example for containerd runtime, docker and crio needs to be not running
+func validateNotActiveRuntimeDisabled(ctx context.Context, t *testing.T, profile string) {
+	disableMap := map[string][]string{
+		"docker":     {"crio"},
+		"containerd": {"docker", "crio"},
+		"crio":       {"docker", "containerd"},
+	}
+
+	expectDisable := disableMap[ContainerRuntime()]
+	for _, cr := range expectDisable {
+		// for example: minikube sudo systemctl is-active docker
+		rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", fmt.Sprintf("sudo systemctl is-active %s", cr)))
```

Comment:
don't we want to be able to test configurations where sysinit package uses openrc implementation instead of systemd?

Reply:
this is for inside minikube ISO/kicbase, this will always be "systemd"